### DISCUSSION TITLE
Will the camera automatically turn off when using the iOS left rear swipe gesture?, I don't know if this is the default behavior, but I hope it shouldn't be closed (before I manually close it)

### DISCUSSION THREAD

**Discussion by JACKZGGG** [capawesome-team/capacitor-mlkit#159] [Q&A]
﻿
Please forgive me for using translation:
As stated in the title, due to some gesture behavior in iOS, the camera inexplicably turned off?
﻿
1. I used startScan to enable it, and 2. At this point, I did not call stopScan. I swiped and held down the left button on iOS, but did not release it, causing the camera to turn off

**Comment by JACKZGGG** [comment]


**Reply by JACKZGGG** [reply]
![z1](https://github.com/capawesome-team/capacitor-mlkit/assets/159008409/4652f266-6c2b-4cf7-bf57-d54ef18d6b37)
![Uploading z3.jpg…]()

**Comment by JACKZGGG** [comment]
If there are any solutions, please let me know. Thank you

**Comment by robingenz** [comment]
Please share a screen recording which shows the current behavior.

**Reply by JACKZGGG** [reply]
[zzzzz.zip](https://github.com/user-attachments/files/15948691/zzzzz.zip)
Can I watch this video

**Reply by robingenz** [reply]
Please post the video directly on GitHub (not as zip file).

**Comment by JACKZGGG** [comment]
https://github.com/capawesome-team/capacitor-mlkit/assets/159008409/9c5c1501-68d2-4c68-a326-fd1088ee40d6

**Reply by robingenz** [reply]
Where is the black screen you mentioned?

**Reply by JACKZGGG** [reply]
Sorry, my description may have caused a misunderstanding. What I mean is, will the camera automatically turn off during the return gesture process

**Reply by robingenz** [reply]
I've never tested that. Does the green dot at the top disappear after a few seconds?

**Reply by JACKZGGG** [reply]
The green dot at the top will not disappear, and this issue only occurs on iOS

**Reply by robingenz** [reply]
Alright, in this case, the camera is not turned off automatically. Feel free to create a bug report and I will take a closer look at it.

**Reply by 726Abhishek** [reply]
> Alright, in this case, the camera is not turned off automatically. Feel free to create a bug report and I will take a closer look at it.

Is this issue already resolved? I'm currently facing the same problem. Once the iPhone light at top turns green, it doesn't change even after closing the scanner.
